### DISCUSSION TITLE
Bunch closes items that are supposed to stay open

### DISCUSSION THREAD

**Discussion by nacohen** [ttscoff/bunch-docs#90] [Q&A]
I have the following commands in my bunch script for processing email:

DEVONthink 3
Mail

%Mail
%DEVONthink 3

When I run the bunch, both DEVONthink 3 and Mail.app open; however, when I select the script again, those apps close. The % symbol does prevent designated apps from closing in other scripts. There is nothing in the log indicating an error.

Any suggestions about what I may be doing incorrectly or is this a bug?

Thanks,

Norm

**Comment by ttscoff** [comment]
Are you including the apps twice in the Bunch, once with % and once without? This may be due to me poorly explaining how the "%" works. A line like "%Mail" does actually launch Mail on its own, it just tells it not to include it when closing. You don't have to call both "Mail" and "%Mail" to accomplish that. It's a bit confusing because with "@" for focusing, you do want to call it once to launch and once to focus.

Bunch _should_ ignore an app when closing if it shows up at _any_ point with a % in the Bunch, so I'll have to look into why it's not, but if you do actually need to call the same app twice for the purpose of the Bunch, add a % each time, it will have the same effect.

**Reply by nacohen** [reply]
Shortly after posting, I did try only using the app names once, prepended with the % sign. That did fix the problem. I hadn't had a time to post a f/u before I received your reply. 

It was unclear to me that you only needed to list the apps once (with or without the % sign). An update to the example script and/or the documentation to make this more clear would likely be helpful to others.

Thank you so much for getting back to me so quickly. I am finding Bunch to be a very useful tool to facilitate contextual computing. Impressive work!

Norm
